scout/server/blueprints/variant/controllers.py

```diff
@@ -365,10 +365,19 @@ def observations(store, loqusdb, case_obj, variant_obj):
         if user_institutes_ids.isdisjoint(other_institutes):
             # If the user does not have access to the information we skip it
             continue
-        document_id = parse_document_id(chrom, str(pos), ref, alt, var_type, case_id)
-        other_variant = store.variant(document_id=document_id)
+
+        other_variant = store.variant(
+            case_id=other_case["_id"], document_id=variant_obj["variant_id"]
+        )
+
+        # IF variant is SV variant, look for variants with different sub_category occurring at the same coordinates
+        if other_variant is None and category == "sv":
+            other_variant = store.case_variant_by_coordinates(
+                other_case["_id"], category, chrom, end_chrom, pos, end
+            )
+
         # If the other variant is not loaded we skip it
-        if not other_variant:
+        if other_variant is None:
```

Comment:
Like here: we should append other case and a set variant None. There is even code for that in the template already:
https://github.com/Clinical-Genomics/scout/blob/df3020ef7a119f7b1f1d76f86c78c73c518c9aa8/scout/server/blueprints/variant/templates/variant/variant_details.html#L183

My current feeling is that the whole issue would be fixed by deleting L380-381 here, but is is probably also good to keep the search-for-different-subtypes part to catch a few more variants exactly?

Reply:
Agree: it makes sense to run a more complex query, but chances are that will also come up negative, so I still think leaving it with case=other_case, variant=None is much better than just hiding the original loqus hit and implicitly pretending the variant was from another institute..

Reply:
I agree, better than nothing!